package.json

```diff
@@ -434,6 +434,12 @@
         }
       ]
     },
+	{
+	  "component": "pill",
+	  "status": "prototype",
```

Comment:
Good call. We should probably keep it prototype until we can get the Pill listbox working with the left and right keys.

Reply:
What's the difference between `prototype` and `prod` in DSR?

Reply:
https://github.com/salesforce/design-system-react/blob/master/package.json#L528

In practice, it's mostly a way to get components in for teams without doing full a11y testing and signals that if you want full a11y, you shouldn't use these.